2021-01-18 16:39:46,293 INFO    MainThread:3799976 [wandb_setup.py:_flush():70] setting env: {}
2021-01-18 16:39:46,293 INFO    MainThread:3799976 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-18 16:39:46,293 INFO    MainThread:3799976 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-18 16:39:46,293 INFO    MainThread:3799976 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedprox/wandb/run-20210118_163946-od1dm38j/logs/debug.log
2021-01-18 16:39:46,293 INFO    MainThread:3799976 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedprox/wandb/run-20210118_163946-od1dm38j/logs/debug-internal.log
2021-01-18 16:39:48,878 INFO    MainThread:3799976 [wandb_run.py:_console_start():1308] atexit reg
2021-01-18 16:39:48,879 INFO    MainThread:3799976 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-18 16:39:48,879 INFO    MainThread:3799976 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-18 16:39:48,879 INFO    MainThread:3799976 [redirect.py:install():205] install start
2021-01-18 16:39:48,880 INFO    MainThread:3799976 [redirect.py:install():220] install stop
2021-01-18 16:39:48,880 INFO    MainThread:3799976 [redirect.py:install():205] install start
2021-01-18 16:39:48,880 INFO    MainThread:3799976 [redirect.py:install():220] install stop
2021-01-18 16:39:48,880 INFO    MainThread:3799976 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-18 16:47:28,991 INFO    MainThread:3799976 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 0
2021-01-18 16:47:28,992 INFO    MainThread:3799976 [wandb_run.py:_restore():1250] restore
2021-01-18 16:47:28,992 INFO    MainThread:3799976 [redirect.py:uninstall():224] uninstall start
2021-01-18 16:47:28,993 INFO    MainThread:3799976 [redirect.py:_stop():279] _stop: stdout
2021-01-18 16:47:28,993 INFO    MainThread:3799976 [redirect.py:_stop():285] _stop closed: stdout
2021-01-18 16:47:28,994 INFO    stdout    :3799976 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-18 16:47:28,994 INFO    stdout    :3799976 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-18 16:47:28,995 INFO    MainThread:3799976 [redirect.py:_stop():291] _stop joined: stdout
2021-01-18 16:47:28,996 INFO    MainThread:3799976 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-18 16:47:28,996 INFO    MainThread:3799976 [redirect.py:uninstall():228] uninstall done
2021-01-18 16:47:28,996 INFO    MainThread:3799976 [redirect.py:uninstall():224] uninstall start
2021-01-18 16:47:28,996 INFO    MainThread:3799976 [redirect.py:_stop():279] _stop: stderr
2021-01-18 16:47:28,996 INFO    MainThread:3799976 [redirect.py:_stop():285] _stop closed: stderr
2021-01-18 16:47:28,997 INFO    stderr    :3799976 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-18 16:47:28,997 INFO    stderr    :3799976 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-18 16:47:28,998 INFO    MainThread:3799976 [redirect.py:_stop():291] _stop joined: stderr
2021-01-18 16:47:28,998 INFO    MainThread:3799976 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-18 16:47:28,998 INFO    MainThread:3799976 [redirect.py:uninstall():228] uninstall done
2021-01-18 16:47:29,500 INFO    MainThread:3799976 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "47278200ba644b7f9cfc930ebbda4aec"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 635
      total_bytes: 635
    }
  }
}

2021-01-18 16:47:31,508 INFO    MainThread:3799976 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "c357b7c32e834b338366d255dd6b5ec4"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 4993
      total_bytes: 4993
    }
  }
}

2021-01-18 16:47:32,801 INFO    MainThread:3799976 [wandb_run.py:_show_files():1614] logging synced files
2021-01-18 16:47:32,802 INFO    MainThread:3799976 [internal.py:handle_exit():139] Internal process exited
